JPS_Ontology/ontology/ontobuiltenv/OntoBuiltEnv.owl

```diff
@@ -1078,9 +1078,9 @@
 
     <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontobuiltenv/Medical">
         <rdfs:subClassOf rdf:resource="https://www.theworldavatar.com/kg/ontobuiltenv/Non-Domestic"/>
-        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Medical include buildings and sites focused on the provision of secondary medical care services or the dispensing of medicinal drugs, or both.</rdfs:comment>
-        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontobuiltenv/</rdfs:isDefinedBy>
-        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Medical</rdfs:label>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Medical include buildings and sites focused on the provision of medical care services or the dispensing of medicinal drugs</rdfs:comment>
+        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string"> or both.</rdfs:isDefinedBy>
```

Comment:
These lines do not look right to me. Something got messed up.

Reply:
Fixed in both owl and csv. Turns out we do need `"` in Medical's comment due to the presence of comma in the sentence.